opentelemetry-prometheus/src/lib.rs

```diff
@@ -1,9 +1,19 @@
 //! An OpenTelemetry exporter for [Prometheus] metrics.
 //!
-//! <div class="warning"> The development of prometheus exporter has halt until the Opentelemetry metrics API and SDK reaches 1.0. Current
-//! implementation is based on Opentelemetry API and SDK 0.23.</div>
+//! <div class="warning">
+//! <strong>Warning: This crate is no longer recommended for use.</strong><br><br>
+//! Development of the Prometheus exporter has been discontinued. See the related 
+//! [issue](https://github.com/open-telemetry/opentelemetry-rust/issues/2451).
+//! This crate depends on the unmaintained `protobuf` crate and has unresolved 
+//! security vulnerabilities. Version 0.29 will be the final release.
+//!
+//! For Prometheus integration, we strongly recommend using the [OTLP] exporter instead.
+//! Prometheus now [natively supports OTLP](https://prometheus.io/docs/guides/opentelemetry/#enable-the-otlp-receiver),
```

Comment:
```suggestion
//! Since version 3, Prometheus [natively supports OTLP](https://prometheus.io/docs/guides/opentelemetry/#enable-the-otlp-receiver),
```

Reply:
I avoided the version mention, because it was available much earlier than 3 - https://github.com/open-telemetry/opentelemetry-demo/issues/1143

It was under experimental feature before, with 3.0 it got stable. Don't think we need to give much details...
(We don't do version mention for jaeger also)